lib/util.js

```diff
@@ -194,6 +194,7 @@ function inspect(obj, opts) {
   if (ctx.colors) ctx.stylize = stylizeWithColor;
   if (ctx.maxArrayLength === undefined) ctx.maxArrayLength = kDefaultMaxLength;
   if (ctx.maxArrayLength === null) ctx.maxArrayLength = Infinity;
+  if (ctx.breakLength === undefined) ctx.breakLength = 60;
```

Comment:
I picked 60 to maintain backwards compatibility. I could assign it to `null` or `Infinity` and then change the value later, but it seems like that would only complicate things.